bundle/src/main/java/com/adobe/acs/tools/csv_asset_importer/impl/CsvAssetImporterServlet.java

```diff
@@ -89,8 +89,30 @@ protected final void doPost(final SlingHttpServletRequest request, final SlingHt
         final Parameters params = new Parameters(request);
         if (params.getFile() != null) {
 
-            final long start = System.currentTimeMillis();
-            final Iterator<String[]> rows = this.getRowsFromCsv(params);
+        	Iterator<String[]> rows = null;
+			final long start = System.currentTimeMillis();
```

Comment:
It's pretty clear you are mixing tabs and spaces here. Please don't do that and respect the original formatting.